### ISSUE TITLE
Relationships between models example

### ISSUE THREAD

**Issue by adamavenir** [HenrikJoreteg/humanjs#8]
http://docs.humanjavascript.com/#human-model-model-relationships says:

"From our experience, `derived` properties work really well for handling relationships between models. Let's say you've got a model representing people and model representing a group of people. Often your API for fetching people would include something like a `groupID` as a property for each person you retrieve."

Having an example of this right below this paragraph would be useful.

**Comment by HenrikJoreteg** [comment]
Yep, agreed. Also we should probably include a sample relationship in the generated app too.
